samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -118,33 +119,47 @@ static bool identifying_info_read_allow(struct bt_conn *conn)
 		return true;
 	}
 
-	LOG_INF("Rejecting read operation of identifying information");
+	LOG_INF("Rejecting operation on the identifying information");
 
 	return false;
 }
 
-static bool gatt_read_authorize(struct bt_conn *conn, const struct bt_gatt_attr *attr)
+static bool gatt_authorize(struct bt_conn *conn, const struct bt_gatt_attr *attr)
```

Comment:
I would improve the `gatt_authorize` function as now we have two modes supported in the following way

```
static bool gatt_authorize(struct bt_conn *conn, const struct bt_gatt_attr *attr)
{
	bool authorize = true;

	if (IS_ENABLED(CONFIG_APP_DFU)) {
		authorize &= app_dfu_bt_gatt_operation_allow();
	}
	authorize &= identifying_info_allow();

	return authorize;
}
```

And then we would move this `for` loop and the uuid_block_list into `identifying_info_allow` function